Client/mods/deathmatch/logic/CClientPlayer.h

```diff
@@ -111,6 +111,9 @@ class CClientPlayer final : public CClientPed
     bool   GetWasRecentlyInNetworkInterruption(uint uiMaxTicksAgo);
     void   SetIsInNetworkInterruption(bool bInNetworkInterruption);
 
+    unsigned long GetPlayerScriptDebugLevel() { return scriptDebugLevel; };
```

Comment:
Why unsigned long when scriptDebugLevel is unsigned int? Make it as const

Reply:
thx, typo